modules/Bio/EnsEMBL/Analysis/Hive/RunnableDB/HiveCreatePlaceholderSQL.pm

```diff
@@ -0,0 +1,112 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl help desk at
+<http://www.ensembl.org/Help/Contact>.
+
+=cut
+
+package Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCreatePlaceholderSQL;
+
+use warnings;
+use strict;
+
+use parent ('Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveBaseRunnableDB');
+
+=head2 fetch_input
+
+ Arg [1]    : None
+ Description: Implements fetch_input() interface method of Bio::EnsEMBL::Hive::Process that is used to read in parameters and load data.
+              Here we have nothing to do.
+ Returntype : None
+ Exceptions : None
+
+=cut
+
+sub fetch_input {
+}
+
+=head2 run
+
+ Arg [1]    : None
+ Description: Creates an sql command add placeholder location information to the meta table
+ Returntype : None
+ Exceptions : None
+
+=cut
+
+sub run {
+  my ($self) = @_;
+
+  my $core_dba = $self->hrdb_get_dba($self->param_required('input_db'));
+  my $sa = $core_dba->get_SliceAdaptor;
+
+  my $sth_longest = $core_dba->dbc->prepare('select seq_region_id from seq_region order by length desc limit 10');
+  $sth_longest->execute;
+
+  my $sample_transcript;
+ LOOP: while (my $seq_region_id = $sth_longest->fetchrow_array) {
+    my $region = $sa->fetch_by_seq_region_id($seq_region_id);
+
+  TRANSCRIPT:foreach my $transcript (@{ $region->get_all_Transcripts_by_type('protein_coding') }){
+      print Dumper $transcript;
```

Comment:
This should be removed, if you want so information, you should use`$self->say_with_header()`. It will only be printed when using `-debug 1`.
There is also too much information in `Dumper`

Reply:
oops left in from testing - removed now